assets/src/js/Components/ArchivePicker.js

```diff
@@ -246,6 +258,20 @@ export default function ArchivePicker() {
       <div className={classNames('image-picker', {'open-sidebar': selectedImagesAmount > 0})}>
         <ArchivePickerList />
 
+        {images.length > 0 && (
```

Comment:
Checking `images.length` is enough!

Reply:
Nope, if I do that a `0` shows when the page is loading

Reply:
Just a comment, but if`images.length` is equal to `0`, `0` will be rendered since you're evaluating with `&&`. Whenever you check for arrays lengths, is better to use ternary conditions `? :`, and return `null` instead.

Reply:
I'm not sure I get why this is better than directly checking `images.length > 0`? Then you don't need the ternary 🤔

Reply:
To avoid getting `0` you can just replace to `!!images.length && ...` to cast it to `false`. This how the [short circuits works](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence#short-circuiting).

Reply:
But take in mind this is not a required change 🙏

Reply:
Updated!